CHANGELOG.md

```diff
@@ -1,13 +1,52 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-06-27
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as v20240627.
+
+### Announcements
+
+* Starting 1.30 Kubernetes version and 1.27 LTS versions, beta APIs will be disabled by default, when you upgrade to [them](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#before-you-begin). There will be an option provided to explicitly enable beta APIs closer to the 1.30 release.
+* 1.30 is the next LTS version after 1.27. Customers will have at least 6 months of migration time available between 1.27 LTS to 1.30 LTS i.e before the 1.27 LTS goes End of Life by July 2025. Refer [long term support](https://learn.microsoft.com/azure/aks/long-term-support) for more information.
+
+### Release Notes
+
+* Features:
```

Comment:
@shashankbarsin can you add the following:
[AKS Cost Analysis](https://learn.microsoft.com/azure/aks/cost-analysis) views can now be accessed directly from the Cluster overview blade in Azure Portal. [Learn more](https://learn.microsoft.com/azure/cost-management-billing/costs/view-kubernetes-costs).